package.json

```diff
@@ -75,14 +75,15 @@
     "jest-cli": "^27.5.0",
     "lerna": "^4.0.0",
     "license-checker": "^25.0.1",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
     "storybook-addon-performance": "^0.16.1",
     "svgo": "^2.0.3",
     "typescript": "^4.6.3",
     "webpack": "^5.70.0"
   },
   "resolutions": {
-    "axe-core": "^4.3.5"
+    "axe-core": "^4.3.5",
+    "@types/react": "^17.0.44"
```

Comment:
Why is this necessary? So we don't use new APIs yet?

Reply:
Added to the PR description:

> Resolve to @types/react^17 in the root package.json: not doing this causes conflicts with third-party dependencies (e.g. https://github.com/s-yadav/react-number-format/issues/630). I'm hoping to remove this in a few months, after the ecosystem has finished migrating